app/services/comment_service.rb

```diff
@@ -10,6 +10,11 @@ def create(post_id, text)
     user.comment!(post, text)
   end
 
+  def find!(comment_id)
+    #TODO check that the user has access to that comment first
```

Comment:
- Layout/LeadingCommentSpace: Missing space after `#`.
- Style/CommentAnnotation: Annotation keywords like `TODO` should be all upper case, followed by a colon, and a space, then a note describing the problem.